go.mod

```diff
@@ -6,46 +6,47 @@ require (
 	github.com/Kong/sdk-konnect-go v0.7.1
 	github.com/kong/go-kong v0.67.0
 	github.com/stretchr/testify v1.11.1
-	k8s.io/api v0.33.4
-	k8s.io/apiextensions-apiserver v0.33.4
-	k8s.io/apimachinery v0.33.4
-	k8s.io/client-go v0.33.4
+	k8s.io/api v0.34.0
+	k8s.io/apiextensions-apiserver v0.34.0
+	k8s.io/apimachinery v0.34.0
+	k8s.io/client-go v0.34.0
```

Comment:
Bumping these will cause what's described in https://github.com/Kong/kubernetes-configuration/pull/600.

This needs to be taken care of first as merging this as is would cause KO compilation to fail when imported.